backend/globaleaks/anomaly.py

```diff
@@ -223,16 +224,18 @@ class Alarm(object):
     }
 
     OUTCOMING_ANOMALY_MAP = {
-        'failed_logins': 50,
-        'successful_logins': 70,
-        'started_submissions': 100,
-        'completed_submissions': 50,
-        'uploaded_files': 100,
-        'appended_files': 100,
-        'wb_comments': 20,
-        'wb_messages': 20,
-        'receiver_comments': 30,
-        'receiver_messages': 30
+         # Remind: started submission at the moment can be triggered also by a crawler
+        'started_submissions': 10,
+        'completed_submissions': 5,
+        'rejected_submissions': 5,
+        'failed_logins': 8,
+        'successful_logins': 3,
+        'uploaded_files': 10,
+        'appended_files': 10,
+        'wb_comments': 2,
+        'wb_messages': 2,
+        'receiver_comments': 6,
+        'receiver_messages': 6
```

Comment:
@vecna ok for this new thresholds.

the only one i will relax for now, is the 'started submission' that i find not critical for the resources that it opens on the system and i would and befor proper tests it will open possible issues for adopters)